user_sync/credentials.py

```diff
@@ -70,6 +70,11 @@ def load_configs(self, connector_type="all"):
         so that credential manager knows which keys and values are needed per file
         """
         root_cfg = ConfigFileLoader.load_root_config(self.root_config)
+        try:
+            console_log_level = root_cfg['logging']['console_log_level'].upper()
+            self.logger.setLevel(console_log_level)
```

Comment:
One thing here - we don't need .upper().  Just use logging.getLevelName(console_log_level) to get the correct name.  Also no need to except valueerror - just key error